src/components/list/List.jsx

```diff
@@ -1,5 +1,79 @@
+import { useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+
+import { changeList } from '../../store/list/listSlice';
+import useListQueries from './listHooks';
+import { fetchRecipe } from '../../store/recipe/recipeSlice';
+
 function List() {
-    return <section>List</section>;
+    const curCuisine = useSelector((state) => state.list?.cuisine);
```

Comment:
{ list } here better to use destruction 
And in the lower case as well

Reply:
this is a typo, it shouldn't be here